pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -3265,6 +3265,19 @@ type CephBlockPoolRadosNamespaceList struct {
 	Items           []CephBlockPoolRadosNamespace `json:"items"`
 }
 
+// CephBlockPoolRadosNamespaceSpecMirroring represents the mirroring configuration of CephBlockPoolRadosNamespaceSpec
+type CephBlockPoolRadosNamespaceSpecMirroring struct {
+	// Enabled is a flag to enable or disable mirroring
+	// +optional
+	Enabled bool `json:"enabled"`
+	// RemotePool is the name of the secondary cluster pool
+	// +optional
+	RemoteNamespace string `json:"remoteNamespace"`
+	// Mode is the mirroring mode
+	// +optional
+	Mode string `json:"mode"`
+}
```

Comment:
namespace also supports the scheduling can you please take care of it.

Reply:
can you link the document, what it means?

I found snapshot based mirroring is this? https://docs.ceph.com/en/quincy/rbd/rbd-mirroring/#create-image-mirror-snapshots

Reply:
```
$ rbd --cluster site-a mirror snapshot schedule add
```
RBD supports adding snapshot scheduling per namespace as well.